experiments/integrated/fluxnet/met_drivers_FLUXNET.jl

```diff
@@ -181,23 +176,24 @@ radiation = ClimaLand.PrescribedRadiativeFluxes(
 )
 
 # Start and end dates of data in MODIS format
-modis_start_date = "A$(Dates.year(UTC_DATETIME[1]))$(lpad(Dates.dayofyear(UTC_DATETIME[1]), 3, "0"))"
-modis_end_date = "A$(Dates.year(UTC_DATETIME[end]))$(lpad(Dates.dayofyear(UTC_DATETIME[end]), 3, "0"))"
-
-MODIS_LAI = single_col_data_matrix(
-    parse_response(
-        check_response(
-            send_get_subset(
-                "MCD15A2H",
-                modis_start_date,
-                modis_end_date,
-                site_ID,
-                band = "Lai_500m",
-            ),
-        ),
-    ),
+modis_start_date = DateTime(
+    "$(Dates.year(UTC_DATETIME[1]))-$(Dates.month(UTC_DATETIME[1]))-$(Dates.day(UTC_DATETIME[1]))T00:00:00.0",
+)
+modis_end_date = DateTime(
+    "$(Dates.year(UTC_DATETIME[end]))-$(Dates.month(UTC_DATETIME[end]))-$(Dates.day(UTC_DATETIME[end]))T00:00:00.0",
 )
 
+MODIS_LAI_path = ClimaLand.Artifacts.find_modis_lai_fluxnet_sites(site_ID)
+MODIS_LAI_raw, header = readdlm(MODIS_LAI_path, ',', header = true)
+dates = DateTime.(MODIS_LAI_raw[:, 1])
+
+# Trim MODIS_LAI to start_date and end_date
+indices = findall(d -> modis_start_date <= d <= modis_end_date, dates)
+filtered_rows = MODIS_LAI_raw[indices, :]
+filtered_dates = dates[indices]
+
+MODIS_LAI = hcat(filtered_dates, filtered_rows[:, 2])
```

Comment:
You could add a comment here explaining why you concatenate them in this way and what the resulting format is